e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -12,6 +12,7 @@
 AWAITING_METRICS = "TensorBoard is awaiting metrics"
 SERVICE_READY = "TensorBoard is running at: http"
 num_trials = 1
+custom_image_for_testing = "custom_alpine"
```

Comment:
what is the `custom_alpine` image?

Also, I recommend using `alpine` because it's tiny and so the test will run quickly... but tensorboard will also fail immediately in that image.  So I would recommend hardcoding `"alpine"` in the test rather than putting it here.  It's a rather useless image for any other purpose.  Also, `custom_image_for_testing` doesn't really explain what the variable represents in the first place.